### PR TITLE
[Profiler] Update ProfilerController.php

### PR DESCRIPTION
**Pull request by error56**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no 
| BC breaks?    | no     
| Deprecations? | no 
| Tests pass?   | yes    
| Fixed tickets | n/a   
| License       | MIT
| Doc PR        | symfony/symfony-docs#...

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php [hunk 1]
```diff
@@ -125,9 +117,7 @@ public function panelAction(Request $request, $token)
      */
     public function toolbarAction(Request $request, $token)
     {
-        if (null === $this->profiler) {
-            throw new NotFoundHttpException('The profiler must be enabled.');
-        }
```
Comment (curry684) [inline src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php]: This one cannot be replaced as it does not disable the profiler after the check!

**Review by curry684 — CHANGES_REQUESTED** [review]
Must revert breaking change

**Comment by curry684** [comment]
Status: needs work

**Review by curry684 — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @error56.

**Pull request merged**